src/screens/Tab/TaskListScreen.tsx

```diff
@@ -1,62 +1,176 @@
+import { TitledCard } from '../../components/TitledCard';
 
+import { compareDates } from '../../services/Calendar/helper_functions';
+
+//TODO remove dummy data
+const tasks = [
+    {
+        church: {
+            churchId: 3,
+            name: 'Elizabeth',
+        },
+        createdAt: '2020-10-04T01:07:23.900Z',
+        date: '2020-08-21T14:30:00.000Z',
+        role: {
+            name: 'RE',
+        },
+        roleId: 4,
+        taskId: 5,
+        user: {
+            firstName: 'Shaun',
+            lastName: 'Tung',
+        },
+        userId: 1,
+    },
+    {
+        church: {
+            churchId: 1,
+            name: 'Hillsborough',
+        },
+        createdAt: '2020-10-04T01:07:23.900Z',
+        date: '2020-08-21T14:30:00.000Z',
+        role: {
+            name: 'AV',
+        },
+        roleId: 1,
+        taskId: 2,
+        user: {
+            firstName: 'Shaun',
+            lastName: 'Tung',
+        },
+        userId: 1,
+    },
+    {
+        church: {
+            churchId: 1,
+            name: 'Hillsborough',
+        },
+        createdAt: '2020-10-04T01:07:23.900Z',
+        date: '2020-08-22T14:30:00.000Z',
+        role: {
+            name: 'AV',
+        },
+        roleId: 1,
+        taskId: 4,
+        user: {
+            firstName: 'Shaun',
+            lastName: 'Tung',
+        },
+        userId: 1,
+    },
+    {
+        church: {
+            churchId: 1,
+            name: 'Hillsborough',
+        },
+        createdAt: '2020-10-04T01:07:23.901Z',
+        date: '2020-08-23T14:30:00.000Z',
+        role: {
+            name: 'AV',
+        },
+        roleId: 1,
+        taskId: 1,
+        user: {
+            firstName: 'Shaun',
+            lastName: 'Tung',
+        },
+        userId: 1,
+    },
+];
+
+const screenWidth = Dimensions.get('window').width;
+
+//TODO ensure real tasks work instead of my dummy tasks
+//use michelle's colors once she sends it over
 export const TaskListScreen = (props: TaskListScreenProps) => {
-    const tasks = useSelector((state) => state.taskReducer.tasks);
-
-    const leftAccessory = () => openDrawerAction(props.navigation.toggleDrawer);
-    const rightAccessory = () => (
-        <TouchableOpacity
-            onPress={() => props.navigation.goBack()}
-            style={{ flexWrap: 'wrap', paddingRight: 10 }}
-        >
-            <Text category={'s1'}> View as</Text>
-            <Text category={'s1'}>Calendar</Text>
-        </TouchableOpacity>
-    );
+    // const tasks = useSelector((state) => state.taskReducer.tasks);
 
-    const renderItem = ({ item }) => (
-        <TaskListItem item={item} navigation={props.navigation} />
-    );
+    //regroups tasks by date
+    const rearrangedTasks = tasks.reduce((acc, task, index) => {
+        if (index === 0) return [[task]];
+        const prevDate = acc[acc.length - 1][0].date;
+        if (compareDates(new Date(prevDate), new Date(task.date))) {
```

Comment:
`compareDates` doesn't exist anymore in this branch
```suggestion
        if (isSameDate(new Date(prevDate), new Date(task.date))) {
```

Reply:
nice catch!